src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_GridView_Items.cs

```diff
@@ -37,41 +37,10 @@ public async Task When_GridViewItems_LayoutSlots()
 			await WindowHelper.WaitForLoaded(gridView);
 			await WindowHelper.WaitForIdle();
 
-			RectAssert.AreEqual(new Rect
-			{
-				X = 0d,
-				Y = 0d,
-				Width = 104d,
-				Height = 104d,
-			},
-			gvi.LayoutSlot);
-
-			RectAssert.AreEqual(new Rect
-			{
-				X = 0d,
-				Y = 0d,
-				Width = 100d,
-				Height = 100d,
-			},
-			gvi.LayoutSlotWithMarginsAndAlignments);
-
-			RectAssert.AreEqual(new Rect
-			{
-				X = 104d,
-				Y = 0d,
-				Width = 104d,
-				Height = 104d,
-			},
-			gvi2.LayoutSlot);
-
-			RectAssert.AreEqual(new Rect
-			{
-				X = 104d,
-				Y = 0d,
-				Width = 100d,
-				Height = 100d,
-			},
-			gvi2.LayoutSlotWithMarginsAndAlignments);
+			RectAssert.AreEqual(new Rect(0, 0, 100, 100), gvi.LayoutSlot);
+			RectAssert.AreEqual(new Rect(0, 0, 100, 100), gvi.LayoutSlotWithMarginsAndAlignments);
+			RectAssert.AreEqual(new Rect(0, 0, 100, 100), gvi2.LayoutSlot);
+			RectAssert.AreEqual(new Rect(0, 0, 100, 100), gvi2.LayoutSlotWithMarginsAndAlignments);
```

Comment:
not sure which makes more sense here? orignal values or 0,0,100x100

when testing against the original xaml from #7578 which this test was created for,
it doesnt seem to produce any problem:
![image](https://user-images.githubusercontent.com/2359550/216472690-28b358d8-5b97-4071-a05a-fafb432fe524.png)

Reply:
Is the corner rounding causing an issue? Or is it something else?

Reply:
Not related to margin/padding or the rounding.

The changes to address the blank LVI issue made the LVI's LayoutSlot (and -With...) to lose the offset relative to the ItemsPresenter. This offset is already included on the ListViewBaseInternalContainer.Frame.Location.
So this should be the expected from now on.